src/BaseTemplates/StarterWeb/Startup.cs

```diff
@@ -39,8 +39,12 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             if (env.IsDevelopment())
             {
-                app.UseDeveloperExceptionPage();
-                app.UseBrowserLink();
+                app.UseDeveloperExceptionPage();$if$ ($platformversion$ == 1.1)
```

Comment:
This corresponds with the versions declared here:

https://github.com/aspnet/Templates/blob/dev/src/Templates.xml#L401-L402